setup.cfg

```diff
@@ -30,7 +30,7 @@ install_requires =
     skyfield-data
     python-dateutil
     lunardate
-    pytz
+    backports.zoneinfo
```

Comment:
I wonder if it's worth making it a [dependency only when needed](https://www.python.org/dev/peps/pep-0508/#environment-markers) or it this will create more problems than it will solve

Reply:
That's a great hint. So something like
```
backports.zoneinfo;python_version<"3.9"
```

Reply:
can you enlight me here? Even if `pytz` is not in the stdlib, it works, doesn't it? And if you want to support Python 3.6 to 3.9, it looks like the only Python version where zoneinfo is in the stdlib is the 3.9, so it's not "widely standard".
In a word or two, why would we have to switch to zoneinfo? Is there a definitive advantage to it?

Reply:
@ewjoachim I updated it and repushed.

Reply:
@brunobord `zoneinfo` uses your system's tz database, while `pytz` *sometimes* brings its own.